client/src/js/components/rates.jsx

```diff
@@ -1,9 +1,85 @@
 import React from 'react';
+// import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { updateQuerySortBy } from '../actions/index';
+import { makeStarElements } from '../../../../helpers/clientHelplers';
```

Comment:
clientHelper is spelled wrong. please update file name

Reply:
good catch!